src/views/InvitationSection.vue

```diff
@@ -91,6 +91,8 @@ export default defineComponent({
   },
   data() {
     return {
+      searchQuery: '',
+      debounce: 0,
```

Comment:
It's not quite clear what is 'debounce' in this context. Maybe 'debounceTimer' would be a better name?